src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -852,23 +867,19 @@ module Types = struct
           ; zkapp_uri
           } =
         let open Option.Let_syntax in
-        let%bind public_key = public_key in
```

Comment:
I'm intrigued; I would have assumed this would fail to compile.

Reply:
nope -- the partial account actually had this be non-optional

Reply:
That's what I thought, I meant I was surprised it worked with these lines gone!